hive/indexer/cached_post.py

```diff
@@ -453,7 +462,9 @@ def _sql(cls, pid, post, level=None):
             ('up_votes',    "%d" % stats['up_votes']),
             ('is_hidden',   "%d" % stats['hide']),
             ('is_grayed',   "%d" % stats['gray']),
-            ('author_rep',  "%f" % stats['author_rep'])])
+            ('author_rep',  "%f" % stats['author_rep']),
+            ('children',    "%d" % min(post['children'], 32767)), # TODO: #115
```

Comment:
what's #115?

Reply:
better question is "what does this have to do with issue #115 :)

Reply:
It has nothing to do with it, just a small mitigation towards that issue. It's been difficult to get hive PR's reviewed in a timely manner and the only alternative is multi-level branches.